app/components/form/instance.hbs

```diff
@@ -0,0 +1,47 @@
+<div {{did-insert this.onInit}}>
+  {{#if this.initialized}}
+    <div class="au-c-main-container">
+      <div
+        class="au-c-main-container__content au-c-main-container__content--scroll"
+      >
+        <div class="au-o-layout">
+          <section class="au-o-region-large">
+            <RdfForm
+              @groupClass="au-o-grid__item"
+              @form={{this.formInfo.form}}
+              @show={{false}}
+              @graphs={{this.formInfo.graphs}}
+              @sourceNode={{this.formInfo.sourceNode}}
+              @formStore={{this.formInfo.formStore}}
+              @forceShowErrors={{true}}
+              @useNewListingLayout={{true}}
+              @level={{2}}
+              class="au-u-max-width-medium"
+            />
```

Comment:
Makes me wonder again if we should create a generic component to avoid code duplication

Reply:
normally these components should guarantee that there are few instances of rdfform being created, i.e. you should use these components as the generic components. Adding another layer of indirection would complicate it too much again imo